### PR TITLE
Account enrichment generic 2.2 

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3716)

## Description
Enrich accounts using one or more integrations.
Supported integrations:
- Active Directory
- SailPoint IdentityNow
- SailPoint IdentityIQ
- PingOne
- Okta
- AWS IAM

Also, the playbook supports the generic command 'iam-get-user' (implemented in IAM integrations. For more information, visit https://xsoar.pan.dev/docs/integrations/iam-integrations. (Available from Cortex XSOAR 6.5.0).

## Screenshots
Account_Enrichment_-_Generic_v2 2

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ssokolovich** [comment]
New PR - [old pr link ](https://github.com/demisto/content/pull/20557)

File: Packs/CommonPlaybooks/ReleaseNotes/2_2_15.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+#### Playbooks
+##### New: Account Enrichment - Generic v2.1
+Enrich accounts using one or more integrations.
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/ReleaseNotes/2_2_15.md:4]: Let's change the RN to something else, we don't need the whole playbook description

File: Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml [hunk 2]
```diff
@@ -0,0 +1,1103 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: e0b7f17c-70f7-4330-8f06-c3729e226adb
+    type: condition
+    task:
+      id: e0b7f17c-70f7-4330-8f06-c3729e226adb
+      version: -1
+      name: Is OKTA v2 Integration Enabled
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml:494]: Remove upper case

File: Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml [hunk 3]
```diff
@@ -0,0 +1,1103 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 0f630de1-9988-461a-883a-6113d3dd6d2a
+    type: condition
+    task:
+      id: 0f630de1-9988-461a-883a-6113d3dd6d2a
+      version: -1
+      name: Is AWS Integration Enabled?
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml:599]: remove uppercase

File: Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml [hunk 4]
```diff
@@ -0,0 +1,1103 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "51":
+    id: "51"
+    taskid: 21d70bb7-8b40-4f49-80ca-b78cb05b24aa
+    type: condition
+    task:
+      id: 21d70bb7-8b40-4f49-80ca-b78cb05b24aa
+      version: -1
+      name: Is SailPoint IdentityNow Integration Enabled?
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml:789]: upper case

File: Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml [hunk 5]
```diff
@@ -0,0 +1,1103 @@
...
+- contextPath: SailPointIdentityNow.Account.systemAccount
+  description: Indicates whether the account is a system account.
+- contextPath: SailPointIdentityNow.Account.uncorrelated
+  description: Indicates whether the account is uncorrelated.
+- contextPath: SailPointIdentityNow.Account.manuallyCorrelated
+  description: Indicates whether the account was manually correlated.
+- contextPath: SailPointIdentityNow.Account.hasEntitlements
+  description: Indicates whether the account has entitlement.
+tests:
+- No tests (auto formatted)
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml:1102]: The test playbook for the original playbook "Account Enrichment - Generic v2.1 - Test"
We can add it here as well

File: Packs/CommonPlaybooks/ReleaseNotes/2_2_15.md (hunk 1 above)
Comment (ssokolovich) [reply Packs/CommonPlaybooks/ReleaseNotes/2_2_15.md:4]: Removed the word `new`

File: Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml (hunk 5 above)
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml:1102]: Correct. Adding it here .

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**